### PR TITLE
P0 wishlist

### PR DESCRIPTION
**Pull request by purvasharmadev**
# Added The wishlist page

kstyle-wishlist-page

### CODE REVIEW

**Review by adityapandadev — APPROVED** [review]

**Review by rutu14 — APPROVED** [review]
The UI looks amazing.

File: product/product.css [hunk 1]
```diff
@@ -77,6 +77,7 @@ input[type=range]::-webkit-slider-thumb {
 ul li {
     font-weight: 600;
     font-size: 14px;
```
Comment (srishtigautam21) [inline product/product.css:79]: use rem here

**Review by srishtigautam21 — APPROVED** [review]
one more thing i would suggest in your  nav-bar is that you can remove the word cart or give the name wishlist as well.So that everything looks consistent

**Comment by purvasharmadev** [comment]
> one more thing I would suggest in your nav-bar is that you can remove the word cart or give the name wishlist as well.So that everything looks consistent

Thank you for the suggestion, I will do it.

File: product/product.css (hunk 1 above)
Comment (purvasharmadev) [reply product/product.css:79]: for values less than 1rem i.e 16px we can use px, i think.

**Comment by purvasharmadev** [comment]
> The UI looks amazing.

Thank you.

**Pull request merged**